src/FileProviders/Manifest.MSBuildTask/src/Manifest.cs

```diff
@@ -11,7 +11,9 @@ namespace Microsoft.Extensions.FileProviders.Embedded.Manifest.Task
 {
     public class Manifest
     {
+#pragma warning disable PUB0001 // Pubternal type IActivator in public API
```

Comment:
This pubternal doesn't matter as much given that it will be compiled into a msbuild task dll. I think keeping the warning disable is fine.

Reply:
Actually on second thought, there's no reason why this needs to be pubternal. I'm going to make it public since this package isn't shipped so there's no harm in exposing the API.